lib/link.js

```diff
@@ -344,6 +344,9 @@ var Receiver = function (session, name, local_handle, opts) {
     if (this.get_option('autoaccept', true)) {
         this.observers.on('message', auto_accept);
     }
+    if (this.get_option('autosettle', true)) {
```

Comment:
Could we change this to:

```
  if (this.local.attach.rcv_settle_mode === 1 && this.get_option('autosettle', true)) {
```
That avoids setting a listener for an event that will never occur and feels a little nicer to me (probably for irrational reasons!). Thanks for this though! Much appreciated!